packages/block-library/src/image/edit.native.js

```diff
@@ -579,5 +588,21 @@ export default compose( [
 			imageSizes,
 		};
 	} ),
+	withDispatch( ( dispatch, { clientId }, { select } ) => {
+		return {
+			wasBlockJustInserted() {
+				const { clearLastBlockInserted } = dispatch( 'core/editor' );
+				const { wasBlockJustInserted } = select( 'core/editor' );
```

Comment:
I am getting a warning about using the string literals to access the data store. I haven't seen a store definition for `core-editor` in other parts of the codebase as yet.

Reply:
I just found the [PR that replaced store names](https://github.com/WordPress/gutenberg/pull/28975) and I saw that the `core/editor` is now `editorStore`

Reply:
I have the changes for this locally. I am encountering the error below. I will look into what could be causing in the morning

<kbd><img src="https://user-images.githubusercontent.com/1509205/109924877-f571ac80-7c8e-11eb-916f-50035d07fe84.png" width="320"></kbd>

Reply:
I didn't see this error pop up for me so you probably have it fixed now. This sounds similar to an issue solved in [this PR](https://github.com/WordPress/gutenberg/pull/25523) where a setting was removed that the mobile side counted on.